src/main/java/com/gmail/woosay333/onlinebookstore/entity/Book.java

```diff
@@ -0,0 +1,41 @@
+package com.gmail.woosay333.onlinebookstore.entity;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.Table;
+import java.math.BigDecimal;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Entity
+@Table(name = "books")
+@Data
```

Comment:
`@Data` is not recommended to use with the `@Entity` because of probable poor query performance